addons/web/controllers/webmanifest.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import json
+import mimetypes
+
+from odoo import http
+from odoo.exceptions import AccessError
+from odoo.http import request
+from odoo.tools import ustr, file_open
+
+
+class WebManifest(http.Controller):
+
+    def _get_shortcuts(self):
+        module_names = ['mail', 'crm', 'project', 'note']
```

Comment:
Any plans to add custom modules to the list?

Reply:
not at the moment, though i have it waiting its time in a deep corner of my mind. don't hold your breath though :)